packages/datadog-plugin-jest/test/jasmine2.spec.js

```diff
@@ -0,0 +1,132 @@
+'use strict'
+const { ORIGIN_KEY } = require('../../dd-trace/src/constants')
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+const {
+  TEST_FRAMEWORK,
+  TEST_TYPE,
+  TEST_NAME: TEST_NAME_TAG,
+  TEST_SUITE: TEST_SUITE_TAG,
+  TEST_STATUS,
+  CI_APP_ORIGIN
+} = require('../../dd-trace/src/plugins/util/test')
+
+describe('Plugin', () => {
+  let jestExecutable
+
+  const jestCommonOptions = {
+    projects: [__dirname],
+    testPathIgnorePatterns: ['/node_modules/'],
+    coverageReporters: [],
+    reporters: [],
+    testRunner: 'jest-jasmine2',
+    silent: true,
+    testEnvironment: 'node'
+  }
+
+  withVersions(plugin, ['jest-jasmine2'], (version, moduleName) => {
+    afterEach(() => {
+      return agent.close()
+    })
+    beforeEach(() => {
+      return agent.load(['jest']).then(() => {
+        jestExecutable = require(`../../../versions/jest@${version}`).get()
+      })
+    })
+    describe('jest with jasmine', function () {
+      this.timeout(60000)
+      it('instruments async and sync tests', function (done) {
+        if (process.env.DD_CONTEXT_PROPAGATION === 'false') return done()
```

Comment:
This is only for tests that are explicitly testing context propagation.

Reply:
oh, ok, will remove then 👍